lib/config/typescript.js

```diff
@@ -1,8 +1,6 @@
 const merge = require('merge');
 
 module.exports = {
-  extends: 'plugin:shopify/esnext',
```

Comment:
I think it makes sense to keep typescript as a base config - it's a foundational tech rather than a library you use.